src/daemon/src/df-daemon/initializer/initializer.go

```diff
@@ -62,51 +65,60 @@ func cleanLocalRepo() {
 		}
 	}()
 	for {
-		select {
-		case <-time.After(time.Minute * 2):
-			func() {
-				log.Info("scan repo and clean expired files")
-				filepath.Walk(G_CommandLine.DFRepo, func(path string, info os.FileInfo, err error) error {
-					if err != nil {
-						log.Warnf("walk file:%s error:%v", path, err)
-					} else {
-						if info.Mode().IsRegular() {
```

Comment:
cc @allencloud I remember you have concerns on this piece of code, so here is the modification. :)